python/src/uagents/config.py

```diff
@@ -43,15 +47,53 @@
 DEFAULT_SEARCH_LIMIT = 100
 
 
+AgentType = Literal["hosted", "local", "mailbox", "proxy", "custom"]
+
+
+class AgentverseConfig(BaseModel):
```

Comment:
Do we actually still need to distinguish between `protocol` and `http_prefix` if we drop support for websockets? To maintain backwards compatibility, we could keep protocol, but make it at least Optional

Reply:
No, we don't need that any more. I was thinking it's not bad to keep it around in case we bring back websockets, but mostly just didn't get around to removing it.

Reply:
realized that it doesn't matter for a user, fine with leaving it for the time being